src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -169,6 +172,13 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bBuiltProjectOutputGroupDependencies%3bGetCopyToOutputDirectoryItems%3bSatelliteDllsProjectOutputGroup%3bPkgdefProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
+    <ProjectReference Include="..\EmbeddedVsix\EmbeddedVsix.csproj">
+      <Name>EmbeddedVsix</Name>
+      <Project>{4BBE7B39-E00A-40FE-88FC-A53C7E49731F}</Project>
+      <VSIXSubPath>EmbeddedVsix</VSIXSubPath>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
The new VSIX project doesn't build an assembly yet. That will be added in a later PR.